source/_components/device_tracker.hitron_coda.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Hitron CODA Routers"
+description: "Instructions on how to integrate Hitron CODA Routers into Home Assistant."
+date: 2017-10-03 15:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: keenetic.png
+ha_category: Presence Detection
+ha_release: 0.55
+---
+
+This component offers presence detection by examining devices connected to a [Rogers Hitron CODA](https://www.rogers.com/customer/support/article/wi-fi-password-hitron-coda4582-cgn3amr-cgnm3552-cgn3acr-cgn3)
+Router.
+
+To use a Hitron router in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: hitron_coda
+    host: !secret router_ip
+    username: !secret router_username
+    password: !secret router_password
+```
+
+Configuration variables:
```

Comment:
Use the new configuration tag.

https://home-assistant.io/developers/documentation/standards/#component-and-platform-pages